src/Builder/Accumulator/FactoryTrait.php

```diff
@@ -0,0 +1,512 @@
+<?php
+
+/**
+ * THIS FILE IS AUTO-GENERATED. ANY CHANGES WILL BE LOST!
+ */
+
+declare(strict_types=1);
+
+namespace MongoDB\Builder\Accumulator;
+
+use MongoDB\BSON\Decimal128;
+use MongoDB\BSON\Document;
+use MongoDB\BSON\Int64;
+use MongoDB\BSON\PackedArray;
+use MongoDB\BSON\Serializable;
+use MongoDB\BSON\Type;
+use MongoDB\BSON\UTCDateTime;
+use MongoDB\Builder\Expression\ResolvesToArray;
+use MongoDB\Builder\Expression\ResolvesToDate;
+use MongoDB\Builder\Expression\ResolvesToInt;
+use MongoDB\Builder\Expression\ResolvesToNumber;
+use MongoDB\Builder\Expression\ResolvesToObject;
+use MongoDB\Builder\Type\ExpressionInterface;
+use MongoDB\Builder\Type\Optional;
+use MongoDB\Model\BSONArray;
+use stdClass;
+
+/**
+ * @internal
+ */
+trait FactoryTrait
+{
+    /**
+     * Defines a custom accumulator function.
+     * New in version 4.4.
```

Comment:
Which version is meant here?

Reply:
It's MongoDB version. It's confusing, I admit.
Better like this?
```suggestion
     * New in MongoDB 4.4.
```

Reply:
Yes